src/software/sensor_fusion/sensor_fusion.cpp

```diff
@@ -40,6 +40,9 @@ std::optional<World> SensorFusion::getWorld() const
     }
 }
 
+unsigned int friendly_goalie_id = 1;
+unsigned int enemy_goalie_id    = 1;
```

Comment:
see below; you can remove these

Reply:
these should be private class members of SensorFusion